### ISSUE TITLE
vcluster create my-vcluster --expose  fatal  Get "https://kubernetes.docker.internal:6443/api/v1/namespaces/vcluster-my-vcluster": dial tcp 127.0.0.1:6443: connect: connection refused

### ISSUE THREAD

**Issue by hughbrien** [loft-sh/vcluster#915]
### What happened?

 I followed the directions. I got the same error on three different commands : 
 
 vcluster create my-vcluster --expose
 vcluster create my-vcluster --isolate 

fatal  Get "https://kubernetes.docker.internal:6443/api/v1/namespaces/vcluster-my-vcluster": dial tcp 127.0.0.1:6443: connect: connection refused

Apple M1 Pro
MacOS Ventura
13.1 (22C65)

### What did you expect to happen?

Cluster was supposed to start

### How can we reproduce it (as minimally and precisely as possible)?

I just rant the command several times. 

### Anything else we need to know?

I turned off docker-desktop 

### Host cluster Kubernetes version

<details>

```console
$ kubectl version
# paste output here
```

</details>

### Host cluster Kubernetes distribution

<details>

```
# Write here
```

</details>

### vlcuster version

<details>

```console
$ vcluster --version
# paste output here
```

</details>

### Vcluster Kubernetes distribution(k3s(default)), k8s, k0s)

<details>

```
# Write here
```

</details>

### OS and Arch

<details>

```
OS: 
Arch:
```
</details>

**Comment by matskiv** [comment]
> fatal Get "https://kubernetes.docker.internal:6443/api/v1/namespaces/vcluster-my-vcluster": dial tcp 127.0.0.1:6443: connect: connection refused
> ...
> I turned off docker-desktop

Yes, so the error is explained by the fact that you turned off docker-desktop - vcluster CLI is not able to connect to the Kubernetes cluster that is configured in kube context.

vcluster is not a direct replacement for docker-desktop/minikube/KinD, it requires a Kubernetes cluster to host it, see [docs introduction](https://www.vcluster.com/docs/what-are-virtual-clusters) and [architecture basics](https://www.vcluster.com/docs/architecture/basics) for more details.
